reana_workflow_controller/k8s.py

```diff
@@ -479,3 +479,16 @@ def delete_dask_dashboard_ingress(cluster_name, workflow_id):
         plural="middlewares",
         name=f"replacepath-{workflow_id}",
     )
+
+
+def check_pod_status_by_prefix(pod_name_prefix, namespace="default"):
```

Comment:
Instead of "default" namespace value, we could use the one we have in `r-commons`, i.e. `REANA_RUNTIME_KUBERNETES_NAMESPACE`.

Reply:
Commit message cosmetics: instead of _"get the session status from..."_ we could say "detect session status from..."

Reply:
Commit scope cosmetics: since the commit touches component named `k8s.py` or `rest/workflows.py`, we should rather use `fix(rest)` instead of `fix(sessions)`.  The scope is usually named after the component or the subcomponent of the source code that the commit touches; it is not named after the functionality. Although we have not been respecting this conventional commits semantics so for so strictly everywhere :wink: